app/mailboxes/support_mailbox.rb

```diff
@@ -82,6 +82,6 @@ def create_contact
   end
 
   def identify_contact_name
-    processed_mail.from.first.split('@').first
+    processed_mail.sender_name
```

Comment:
If there is no name in the email, then the system auto-generates a random name for the user. To avoid that, please provide the first line as a fallback. 

<img width="353" alt="Screenshot 2021-09-15 at 6 25 34 PM" src="https://user-images.githubusercontent.com/2246121/133437183-9ff9d904-c37c-4e90-ac1a-ca816e1e1b88.png">

Reply:
oh o! Yeah 👍🏼